pkg/utils/utils.go

```diff
@@ -344,7 +348,16 @@ func GetS3PETMFromParamsMap(params map[string]interface{}, logger logrus.FieldLo
 	// If the credentials are explicitly provided in params, use it.
 	// else let aws API pick the default credential provider.
 	var sess *session.Session
-	if _, ok := params[constants.AWS_ACCESS_KEY_ID]; ok {
+	var sessionOptions session.Options
+	caCert, ok := GetStringFromParamsMap(params, "caCert", logger)
+	if ok {
```

Comment:
I got confused here. Do you mean the Region field in aws.Config will only be initialized if there is `caCert` configuration? That doesn't sound correct.

Reply:
Remove the if bracket, Region field in aws.Config should be initialized here without any conditions.